### ISSUE TITLE
Пересчёт суммы в корзине, при изменении курса валюты.

### ISSUE THREAD

**Issue by sanderrr** [abricos/abricos-mod-eshopcart#1]
На данный момент сумма в корзине и цена товаров в корзине не обновляется просле пересчета стоимости товара по другому курсу.

**Comment by roosit** [comment]
Скорее всего цену пересчитывать и не нужно, потому как пользователь, положивший товар по одной цене, может вдруг увидеть при выписывания счета, другую цену. А это может в свою очередь вызвать массу недовольства. 
Думаю, будет рациональнее, на отдельной странице информации для покупателей или в договоре оферты указать, что в связи с острым колебанием курса валют, цена при оформлении товара может меняться.

**Comment by sanderrr** [comment]
Да, если пользователь только что добавил товар и тут резко поменялась цена при оформлении, согласен, но есть нюанс - как я понял наличие товара в корзине сохраняется в куки пользователя и при следующем заходе на сайт товар никуда не исчезает. Возникает вопрос - как быть, если товар был добавлен неделю назад? Как мне кажется, лучше все-же пересчёт + информирование в корзине о том, что цена пересчитана, хотя можно ограничится и простым информированием.
